src/test/scala/com/fortysevendeg/android/sqlite/statement/SQLDroidStatementSpec.scala

```diff
@@ -0,0 +1,439 @@
+package com.fortysevendeg.android.sqlite.statement
+
+import java.sql.{ResultSet, SQLException}
+
+import android.database.Cursor
+import com.fortysevendeg.android.sqlite.logging.LogWrapper
+import com.fortysevendeg.android.sqlite.resultset.SQLDroidResultSet
+import com.fortysevendeg.android.sqlite.{TestLogWrapper, SQLDroidConnection}
+import com.fortysevendeg.android.sqlite._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/811484974/issues/source?bid=2442814&fileBranchId=2740117#l9)